server/platform-support/platform-support.rst

```diff
@@ -108,127 +108,63 @@
      - |checkmark|
      - |checkmark|
 
-   * - Oracle Linux (Red Hat Compatible Kernel only) 9.0+
+   * - RHEL/CentOS Stream/Rocky/Alma/Oracle Linux 9.0+
```

Comment:
[question for review]

To reduce the number of rows in this table, I combined the Oracle Linux rows into the RHEL rows for the corresponding versions. When we do this, we lose the "Red Hat Compatible Kernel only" blurb. Is this acceptable, or does Oracle Linux need to stay in its own row?

Reply:
I think its fine to combine, however is there a way for us to add a note below the table or somewhere for us to keep the blurb. It is an important distinction.

Reply:
Added a footnote at the bottom of the table